### PR TITLE
Let the specified extension(s) override (and extend) the built-in ones

### PR DESCRIPTION
**Pull request by vaygr**
This changes order for built-in tuple of markdown extensions and extensions passed on the commandline allowing to work the latter properly.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Markdown
 Pygments
-bottle
\ No newline at end of file
+bottle
+pymdown-extensions
```
Comment (vaygr) [inline requirements.txt]: But I don't see where would we use `pymdownx.pymdown`, no?

Comment (ncornette) [inline requirements.txt]: I'm sorry, it was the wrong link, according to https://facelessuser.github.io/pymdown-extensions/ it is ok ! sorry for the confusion...

Comment (vaygr) [inline requirements.txt]: Cool, we're good to go then.

**Comment by vaygr** [comment]
@ncornette I believe we're good to merge?

**Comment by ncornette** [comment]
Yeah, sorry for the delay, I will test and merge soon

**Pull request merged**